src/olympia/abuse/models.py

```diff
@@ -235,6 +227,21 @@ def notify_reporters(self, action_helper):
             is_appeal=True,
         )
 
+    def handle_escalate_action(self, *, notes):
+        entity_helper = self.get_entity_helper(
+            self.target,
+            addon_version_string=None,
+            resolved_in_reviewer_tools=True,
+        )
+        self.notes = notes
+        job_id = entity_helper.workflow_recreate(job=self)
+        self.update(
+            job_id=job_id,
+            notes=notes,
+            resolvable_in_reviewer_tools=True,
+            is_forwarded=True,
+        )
```

Comment:
As discussed on slack, this feels wrong. We're potentially creating a new job in Cinder, but in AMO losing the reference to the old one. If there was anything linked to the old job AMO now see it as linked to the new one... 

In addition, this doesn't handle the fact that Cinder could be re-using an existing job, so we could end up failing some unique constraint. I think the proper way of doing this is to close the old job before opening the new one (maybe in `workflow_recreate` itself?).

Also... this should all be done in a celery task since it's going to call Cinder API.